assets/src/components/propertiesPanel/crowdingDiagram.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react"
+import { crowdingIcon } from "../../helpers/icon"
+import { Vehicle } from "../../realtime"
+
+const extendedOccupancyStatus = (vehicle: Vehicle): string => {
+  // GTFS-RT has an EMPTY occupancy status but we don't use it; an empty
+  // vehicle will have a MANY_SEATS_AVAILABLE status. Additionally, it's
+  // possible that the route this vehicle is on generally has crowding data,
+  // but it's not working at the moment for this vehicle in particular.
+
+  if (vehicle.load === null) {
+    return "NO_DATA"
+  }
+
+  if (vehicle.load === 0) {
+    return "EMPTY"
+  }
+
+  // If vehicle.load isn't null, vehicle.occupancyStatus shouldn't be either.
+  return vehicle.occupancyStatus!
+}
+
+const GenericCrowdingDiagram = ({
+  vehicle,
+  statusDescription,
+}: {
+  vehicle: Vehicle
+  statusDescription: string
+}) => {
+  const classModifier = statusDescription.toLowerCase().replace(" ", "-")
```

Comment:
I generally don't like taking human readable strings ("Some crowding") and turning them into machine-readable strings ("some-crowding") because at the string literal it's not clear that some other code (the css selector) is going to match against it. In the future we might decide that "Medium crowding" is better microcopy and break the css.

What if you did (roughly, I know I'm misusing js switch statements here)
```
    case "FEW_SEATS_AVAILABLE":
      return <GenericCrowdingDiagram vehicle={vehicle} status="FEW_SEATS_AVAILABLE"/>
```
and then in this component
```
statusDescription = switch status {
  case "FEW_SEATS_AVAILABLE": "Some crowding"
}
classModifier = switch status {
  case "FEW_SEATS_AVAILABLE": "some-crowding"
}
```